index.rst

```diff
@@ -1,5 +1,6 @@
 A-Team Bootcamp
 ===============
+ .. image:: https://wiki.mozilla.org/images/thumb/1/13/Ateam.png/173px-Ateam.png
```

Comment:
This is probably a worthwhile change also, but could you put it in a separate commit / PR?